lib/routes/route.js

```diff
@@ -612,6 +612,35 @@ module.exports = function(app, router) {
         });
     });
 
+    router.get("/user", function (req, res, next) {
```

Comment:
Yes it would be good. 
Will this flow be good for testing:
To check if the user is logged in, we first need to add a user, similar to `beforeEach` of `check_username_integ_test` and then login the user, similar to `performUserLogin`. Finally testing `/user` route using ` status` or `errorID` for assertion.
 
PS: Latest commit is just merged commit, pushed it by mistake.

Reply:
Yup that sounds like a good flow to me. 👍